controllers/pkg/reconcilers/bootstrap-packages/reconciler.go

```diff
@@ -89,144 +82,135 @@ func (r *reconciler) SetupWithManager(ctx context.Context, mgr ctrl.Manager, c a
 type reconciler struct {
 	client.Client
 	porchClient client.Client
-
-	l logr.Logger
+	log         logr.Logger
 }
 
 func (r *reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	r.l = log.FromContext(ctx)
-	cr := &porchv1alpha1.PackageRevision{}
-	if err := r.Get(ctx, req.NamespacedName, cr); err != nil {
+	r.log = log.FromContext(ctx)
+	packageRevision := &porchv1alpha1.PackageRevision{}
+	if err := r.Client.Get(ctx, req.NamespacedName, packageRevision); err != nil {
```

Comment:
We are discussing this in another thread but this means we have to change the code for every controller rather than just copying things. When I started I did this, but if you write many controllers you get bored by it. This is DDD.

Reply:
I take your point but for me readability is more important. As a human, who is much more prone to errors, code should be easy to follow. In this context, the var name may be obvious to most, but I don't see the gain in not naming it descriptively.

Reply:
ok we differ in opinion here

Reply:
so here is my suggestion, if you want to debate this put this in another PR and we can debate it there. In general it is better to do a PR that changes 1 thing iso multiple things, since you see that you might get stuck on certain aspects.

Reply:
Done